pages/view-plans/index.js

```diff
@@ -0,0 +1,7 @@
+export default function AllPlansView(){
+    return (
+        <>
+        <div>Deploy deploy deploy</div>
+        </>
```

Comment:
you don't need fragments here:

```suggestion
        <div>Deploy deploy deploy</div>
```